.github/workflows/build.yml

```diff
@@ -0,0 +1,324 @@
+name: Build
+on:
+  push:
+    branches:
+      - master
+      - branch-*
+      - dogfood-*
+  pull_request:
+  merge_group:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: ${{ github.ref_name != github.event.repository.default_branch }}
+
+jobs:
+  build:
+    runs-on: sonar-l-public
```

Comment:
Why `sonar-l-public`? `sonar-m-public` is already 10 CPU and 32 Gb, previous config on CirrusCI mention 2 CPU and 4 Gb for build only.

Reply:
Ok moved to `sonar-m-public`. I keep only `sonar-l-public` for ruling. The DevXp is planning to introduce observability so then we will know better what machine is needed.